### PR TITLE
reenable poapz 

### PR DESCRIPTION
**Pull request by skylarbarrera**
we had some perf issues with rendering gif previews, we now show png's and the full assets only on the expanded state. also disabled some UI elements that are only for mainnet/opensea assets.

PoW: https://cloud.skylarbarrera.com/Screen-Recording-2021-11-29-10-22-02.mp4 

also fixed the text for iOS photo permissions
- before: https://cloud.skylarbarrera.com/Screen-Shot-2021-11-28-15-30-07.53.png
- after: https://cloud.skylarbarrera.com/Screen-Shot-2021-11-28-15-29-54.44.png

QA: make sure scrolling isn't choppy when displaying poaps, confirm that all pricing/opensea specific UI elements are disabled for xDai poaps

### CODE REVIEW

File: src/components/expanded-state/unique-token/UniqueTokenExpandedStateContent.js [hunk 1]
```diff
@@ -64,8 +64,9 @@ const UniqueTokenExpandedStateContent = ({
     if (asset.image_url?.startsWith?.(GOOGLE_USER_CONTENT_URL) && size > 0) {
       return `${asset.image_url}=w${size * MAX_IMAGE_SCALE}`;
     }
+    if (asset?.isPoap) return asset.animation_url;
     return asset.image_url;
-  }, [asset.image_url, size]);
+  }, [asset.animation_url, asset.image_url, asset?.isPoap, size]);
```
Comment (brunobar79) [inline src/components/expanded-state/unique-token/UniqueTokenExpandedStateContent.js:69]: why `asset?.isPoap` if no one else is checking ?

**Review by brunobar79 — DISMISSED** [review]
💯

File: src/components/expanded-state/unique-token/UniqueTokenExpandedStateContent.js (hunk 1 above)
Comment (skylarbarrera) [reply src/components/expanded-state/unique-token/UniqueTokenExpandedStateContent.js:69]: funky, approve again pls 😘

**Review by brunobar79 — APPROVED** [review]

**Review by brunobar79 — APPROVED** [review]

**Review by osdnk — APPROVED** [review]
anything blocking?

**Review by ibrahimtaveras00 — APPROVED** [review]
👍🏽

**Pull request merged**